mobile/lib/features/trade/position_list_item.dart

```diff
@@ -45,10 +48,18 @@ class _PositionListItemState extends State<PositionListItem> {
     formatter.maximumFractionDigits = 2;
 
     TradeTheme tradeTheme = Theme.of(context).extension<TradeTheme>()!;
+    PositionChangeNotifier positionChangeNotifier = context.watch<PositionChangeNotifier>();
+
+    // We're a bit conservative, we only enable action when we have both bid and ask
```

Comment:
I think this comment is at the wrong line, given that we set `hasPriceInOrderBook` at line `59`